stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -21,8 +22,8 @@ internal class PaymentOptionsViewModel(
     config = args.config,
     prefsRepository = prefsRepository
 ) {
-    private val _userSelection = MutableLiveData<PaymentSelection>()
-    val userSelection: LiveData<PaymentSelection> = _userSelection
```

Comment:
I now don't remember why we have `selection` in `SheetViewModel` and `userSelection` here. They definitely seem redundant based on usage.

Reply:
This was my rabbit hole yesterday.  THe userSelection was used for returning the item selected by the user.  Selection is updated every time data in the fragment can be validated (not necessarily selected)